net-im/discord-bin/Manifest

```diff
@@ -1 +1,2 @@
 DIST discord-0.0.5.deb 52100102 BLAKE2B d8b28386d83fd63e47f11af06bc9858a3877565785c97588dad0ade197d70a8d3635137a66024a3b82c4c65acc6e806981a2d9e0e5f20b17745d9123a2cf5d4a SHA512 8b2ef353d393644e450452060449c633487ff344f2e9e0711029bbb3012c364211edc30507976ec218f8a573aefd603ea1681b4e37fe9561f39c5eaa5fd6e29a
```

Comment:
You should probably make a 2nd commit and drop 0.0.5 since it's unfetchable anyway.

Reply:
Done